### DISCUSSION TITLE
打开文件报错

### DISCUSSION THREAD

**Discussion by xiaojia21190** [nshen/learn-neovim-lua#127] [Q&A]
E5108: Error executing lua: ...nvim-tree.lua/lua/nvim-tree/actions/node/system-open.lua:39:
ENOENT: no such file or directory
NvimTree system_open: failed to spawn process using 'open'.
stack traceback:
        [C]: in function 'error'
        ...nvim-tree.lua/lua/nvim-tree/actions/node/system-open.lua:39: in function 'handle_tree_actions'
        ...r/start/nvim-tree.lua/lua/nvim-tree/actions/dispatch.lua:120: in function 'dispatch'
        ...acker/start/nvim-tree.lua/lua/nvim-tree/actions/init.lua:260: in function <...acker/start/nvim-tree.lua/lua/nvim-tree/actions/init.lua:259>

**Comment by nshen** [comment]
这个是 nvim-tree 的 调用系统打开文件，如按我的配置应该是按了 Enter 键，这个跟系统有关，wsl下记得要装 wsl-open，也可能是系统不能识别该文件类型。 正常我在 mac下按 Enter 会用vscode打开文本文件。

如果想用neovim打开文件按 o 就可以。

**Reply by xiaojia21190** [reply]
ok, 已解决, 改变了一下快捷键
